cmd/maven-redhat/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"flag"
+	"log"
+
+	"github.com/openshift-knative/hack/pkg/maven"
+)
+
+const (
+	pomFileFlag = "path"
+)
+
+func main() {
+	metadata, err := maven.ScrapRedHatMavenRegistry(maven.RedHatMavenGA)
```

Comment:
If it's only Quarkus version, we might be good with:
```
➜  ~ curl -s "https://code.quarkus.redhat.com/api/platforms" | jq -r '.platforms[0].streams[0].releases[0].version'
3.8.5.SP1-redhat-00001
```

Reply:
SP1 is actually newer than `3.8.5.redhat-00003`. Where regular 3.8.5 is standard release. Then SP1 stands for s' security patch 1 of 3.8.5 I believe.

https://maven.repository.redhat.com/ga/com/redhat/quarkus/platform/quarkus-bom/3.8.5.SP1-redhat-00001/

Reply:
SP == service patch, sorry :)

Reply:
hmm, ok, I think this is already better than what we currently have in Konflux builds, so special sorting can be added later or we can always add "bump dependencies" using other tools later (like renovate or dependabot)